build/Dockerfile.devel

```diff
@@ -1,4 +1,4 @@
-FROM foundationdb/foundationdb-build:0.1.17
+FROM foundationdb/foundationdb-build:0.1.18
```

Comment:
Should I revert the changes to build/Dockerfile.devel and build/docker-compose.yaml until 0.1.18 is published?

Reply:
The normal process is to rebase this to 6.2.  At that point, we'll build + push a new docker.  That way, this can be done in a single PR.